lib/tasks/auto_annotate_models.rake

```diff
@@ -1,35 +0,0 @@
-# NOTE: only doing this in development as some production environments (Heroku)
```

Comment:
The deletion of this file is mostly unrelated. Basically, I think it wasn't doing anything (for our workflows).